playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -24,6 +24,8 @@
       openshift_node_image_prep_packages:
       - strace
       - tcpdump
+      - vim-enhanced
+      - git
```

Comment:
why git?  That seems like something that should be in a container, not the image.  vim seems ok to me for someone poking around.

Reply:
I wanted to git clone from github, modify it, and deploy something. It was more for development.